qiskit_metal/_gui/widgets/library_new_qcomponent/parameter_entry_window.py

```diff
@@ -11,52 +11,77 @@
 # Any modifications or derivative works of this code must retain this
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
+
+# pylint: disable-msg=too-many-return-statements
+# pylint: disable-msg=no-else-return
+# pylint: disable-msg=inconsistent-return-statements
+# pylint: disable-msg=attribute-defined-outside-init
+# pylint: disable-msg=broad-except
+# pylint: disable-msg=import-outside-toplevel
+# pylint: disable-msg=relative-beyond-top-level
+# pylint: disable-msg=import-error
+# pylint: disable-msg=too-few-public-methods
+# pylint: disable-msg=too-many-instance-attributes
+# pylint: disable-msg=invalid-name
+# pylint: disable-msg=no-name-in-module
```

Comment:
Oh wow, should we maybe disable some of these globally, rather that at file level? This seem to be quite the lengthy list.
Note deleting those altogether could be an option that does not hinder CI/CD. Yapf only complained about about line 459 (previous 420).

I am good either way, we can keep them now and try to remember to remove them later before running the global linting cleanup.

Reply:
Quite possibly, but that's out of scope for this PR.